trl/extras/best_of_n_sampler.py

```diff
@@ -21,6 +21,29 @@
 
 
 class BestOfNSampler:
+    """Initialize the sampler for best-of-n generation
+
+    Args:
+        model (`PreTrainedModelWrapper`):
+            The pretrained model to use for generation
+        tokenizer (`PreTrainedTokenizer` or `PreTrainedTokenizerFast`):
+            Tokenizer associated with the pretrained model
+        queries_to_scores (`Callable[[list[str]], list[float]]`):
+            Callable that takes a list of generated texts and returns the associated reward scores
+        length_sampler (`Any`):
+            Sampler used to sample the length of the generated text
+        sample_size (`int`):
+            Number of samples to generate for each query
+        seed (`int`, *optional*):
```

Comment:
```suggestion
        seed (`int` or `None`, *optional*, default to `None`):
```

Reply:
I can make a first pass with the evident corrections and a second pass with this one...

Reply:
I'll answer here  #4036